### ISSUE TITLE
A WPA 2 karma attack (saved network handshaker ) 

### ISSUE THREAD

**Issue by minanagehsalalma** [meloluan/espRogue#1]
as karma attacks works on saved open networks only and not secured ones so we will need to have the password first to get it work although this seems a bit hard... the number of saved networks Guarantee that there is a one with a weak password... so will need to make the karma attack start two networks of the targeted prob one open and the second secured and the victim will auto connect to the one with the correct configuration with the saved obviously correct password and we save that handshake with the other handshakes captured form the same device to brute force and break the ones with the weak passwords.

**Comment by meloluan** [comment]
Exactly, I am studying if this attack in an ESP8266 is really possible, probably not. Maybe I'll change the focus to an Evil Twin Attack or something, just like Fluxion.

**Comment by minanagehsalalma** [comment]
> Exactly, I am studying if this attack in an ESP8266 is really possible, probably not. Maybe I'll change the focus to an Evil Twin Attack or something, just like Fluxion.

Great if so check this 
https://github.com/skickar/Esp8266Wpa2Handshake/issues/1
Evil twin on two esps

**Comment by meloluan** [comment]
I'll follow the discussion, but I didn't want there to be a need for two ESP8266

**Comment by minanagehsalalma** [comment]
> I didn't want there to be a need for two ESP8266

i think that's the only way to do it !

**Comment by meloluan** [comment]
Take a look at this: https://github.com/senorblasto/fluxion-esp8266

**Comment by minanagehsalalma** [comment]
> Take a look at this: https://github.com/senorblasto/fluxion-esp8266

I have did a long time ago ... It only launchs the fake app and does noting else !
